apps/vite/src/pages/UpcomingEvents.tsx

```
@@ -1,14 +1,17 @@
-import NepaliDate from "nepali-datetime"
-import SingleCalendarEvent from "../components/SingleCalendarEvent"
-import { DayData } from "@miti/types"
+import NepaliDate from 'nepali-datetime';
+
+import { DayData } from '@miti/types';
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)